ChangeLog.md

```diff
@@ -36,9 +41,8 @@ Current Trunk
   compiler options.
 - Allow spaces in a path to Python interpreter when running emscripten from Unix
   shell (#11005).
-- Remove `BINARYEN_PASSES` settings.  We still have `BINARYEN_EXTRA_PASSES`, but
-  completely overriding the set of passes from the command line didn't make much
-  sense.
+- Support atexit() in standalone mode (#10995). This also fixes stdio stream
+  flushing on exit in that mode.
```

Comment:
ditto

Reply:
Same, it just wasn't in the PR, but I think it's worth mentioning.